sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2023-2024.bpmn

```diff
@@ -95,84 +93,212 @@
             <bpmn:flowNodeRef>Event_1bplh1d</bpmn:flowNodeRef>
             <bpmn:flowNodeRef>Activity_134pojh</bpmn:flowNodeRef>
             <bpmn:flowNodeRef>Activity_0qzkp53</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_1sg75ph</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_15d88qb</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Gateway_0pzv80g</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_10tndoy</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_0wg31nu</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Gateway_0rrzb1g</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Gateway_1gcbtrb</bpmn:flowNodeRef>
             <bpmn:flowNodeRef>Event_0re3zsg</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Gateway_1blaf95</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_0xpdeqq</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_1e56skq</bpmn:flowNodeRef>
+            <bpmn:flowNodeRef>Event_1dvh1rf</bpmn:flowNodeRef>
             <bpmn:flowNodeRef>Event_1j6ep9h</bpmn:flowNodeRef>
-            <bpmn:flowNodeRef>Event_1pr211h</bpmn:flowNodeRef>
           </bpmn:lane>
         </bpmn:childLaneSet>
       </bpmn:lane>
     </bpmn:laneSet>
-    <bpmn:intermediateThrowEvent id="Event_123oime" name="calculatedDataTotalEligibleDependants">
+    <bpmn:sequenceFlow id="Flow_19cfg17" sourceRef="Gateway_1blaf95" targetRef="Event_1j6ep9h" />
+    <bpmn:sequenceFlow id="Flow_0wfngj4" sourceRef="Event_1dvh1rf" targetRef="Gateway_1blaf95" />
+    <bpmn:sequenceFlow id="Flow_04xt4on" sourceRef="Event_1e56skq" targetRef="Gateway_1blaf95" />
+    <bpmn:sequenceFlow id="Flow_173rbte" sourceRef="Event_0xpdeqq" targetRef="Gateway_1blaf95" />
+    <bpmn:sequenceFlow id="Flow_0mlhznt" name="Offering weeks less than or equals 17" sourceRef="Gateway_1gcbtrb" targetRef="Event_1dvh1rf">
```

Comment:
Maybe we can avoid repeating the logic in the below calculations and have one more variable created to store 17, 34, or 52.

![image](https://github.com/bcgov/SIMS/assets/61259237/73d7980b-00f2-4cad-a0d4-49c38488f154)